ci/verify-sasl.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
 set -eux
-# TODO
-return 1
+
+# Requires STARTTLS to be active.
+# Requires the cert to be trusted by the host that this is run on.
+echo -e "AUTH LOGIN\ntest\ntest\nQUIT" | openssl s_client -starttls smtp -crlf -quiet -connect 10.99.1.12:25 | grep ^334
```

Comment:
For some reason, private IPs are considered sensitive. Let's not hard-code this--we can add it to the pipeline or extract from terraform outputs.